src/test/scala/io/joern/javasrc2cpg/querying/CfgTests.scala

```diff
@@ -39,7 +39,7 @@ class CfgTests extends JavaSrcCodeToCpgFixture {
   }
 
   "should find that println post dominates correct nodes" in {
-    cpg.call("println").postDominates.size shouldBe 6
+    cpg.call("println").postDominates.size shouldBe 7
```

Comment:
This changed due to string literals being added, in this case the `"foo"` argument to println.